device-types/Cisco/AIR-AP2802I-Q-K9.yaml

```diff
@@ -0,0 +1,14 @@
+---
+manufacturer: Cisco
+model: Aironet 2802I-Q
+slug: cisco-air-ap2802i-q-k9
+part_number: AIR-AP2802I-Q-K9
+u_height: 0
+is_full_depth: false
+comments: '[Cisco Aironet 2800 Series Access Points Data Sheet](https://www.cisco.com/c/en/us/products/collateral/wireless/aironet-2800-series-access-points/datasheet-c78-736497.html)'
+interfaces:
+  - name: WAN
+    type: 1000base-t
+    poe_mode: pd
+  - name: WiFi
+    type: ieee802.11ac
```

Comment:
Same comment about interface naming and PoE spec

Reply:
Where do I find the WLAN interface names?
These names were based on other available `Aironet` device_types in the master branch

Reply:
Interface names, fixed.